app/models/edition.rb

```diff
@@ -212,6 +212,10 @@ def substitute?
     unpublished? && unpublishing.substitute?
   end
 
+  def draft?
+    state == "draft"
```

Comment:
I'd dare say content_store == "draft" is our de-facto method rather than state, what do you think?